CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+### 1.1.1
+* changed `devops` tag to `shallow` while keeping `devops` for backward compatibility
```

Comment:
This can be reworded as Introducing `shallow` tag, since `devops` has been there since day 1, user wouldn't think that it will disappear.
We can add the note about retiring the `devops` tag in later version.